aws/rds_enhanced_monitoring/tests/test_rds_metrics.py

```diff
@@ -0,0 +1,268 @@
+            "rrqmPS": 0,
+            "util": 0.09,
+            "avgQueueLen": 0.02,
+            "tps": 9.47,
+            "readKb": 4,
+            "device": "nvme5n1",
+            "writeKb": 17020,
+            "avgReqSz": 119.89,
+            "wrqmPS": 3.07,
+            "writeIOsPS": 9.43
+        }, {
+            "writeKbPS": 576.53,
+            "readIOsPS": 0,
+            "await": 2.64,
+            "readKbPS": 0,
+            "rrqmPS": 0,
+            "util": 0.09,
+            "avgQueueLen": 0.02,
+            "tps": 9.8,
+            "readKb": 0,
+            "device": "nvme2n1",
+            "writeKb": 17296,
+            "avgReqSz": 117.66,
+            "wrqmPS": 3.9,
+            "writeIOsPS": 9.8
+        }],
+        "processList": [{
+            "vss": 11170084,
+            "name": "aurora",
+            "tgid": 8455,
+            "parentID": 1,
+            "memoryUsedPc": 66.93,
+            "cpuUsedPc": 0.00,
+            "id": 8455,
+            "rss": 10487696
+        }, {
+            "vss": 11170084,
+            "name": "aurora",
+            "tgid": 8455,
+            "parentID": 1,
+            "memoryUsedPc": 66.93,
+            "cpuUsedPc": 0.82,
+            "id": 8782,
+            "rss": 10487696
+        }, {
+            "vss": 11170084,
+            "name": "aurora",
+            "tgid": 8455,
+            "parentID": 1,
+            "memoryUsedPc": 66.93,
+            "cpuUsedPc": 0.05,
+            "id": 8784,
+            "rss": 10487696
+        }, {
+            "vss": 647304,
+            "name": "OS processes",
+            "tgid": 0,
+            "parentID": 0,
+            "memoryUsedPc": 0.18,
+            "cpuUsedPc": 0.02,
+            "id": 0,
+            "rss": 22600
+        }, {
+            "vss": 3244792,
+            "name": "RDS processes",
+            "tgid": 0,
+            "parentID": 0,
+            "memoryUsedPc": 2.80,
+            "cpuUsedPc": 0.78,
+            "id": 0,
+            "rss": 441652
+        }]
+    }
+"""
+
+
+class TestRDSEnhancedMetrics(unittest.TestCase):
+    def test_extract_json_objects(self):
+        # Basic JSON
+        input_string = """{"a":2}{"b":3}"""
+        output_list = ['{"a":2}', '{"b":3}']
+        self.assertEqual(extract_json_objects(input_string), output_list)
+
+        # JSON including brackets
+        input_string = """{"a":2}{"b":"{}{}"}"""
+        output_list = ['{"a":2}', '{"b":"{}{}"}']
+        self.assertEqual(extract_json_objects(input_string), output_list)
+
+        # Highly nested JSON
+        input_string = """{"a":2}{"b":{"c":{"d":{"e":{"f":2}}}}}"""
+        output_list = ['{"a":2}', '{"b":{"c":{"d":{"e":{"f":2}}}}}']
+        self.assertEqual(extract_json_objects(input_string), output_list)
+
+        # JSON with AWS example
+        input_string = full_message_example.replace(" ", "").replace("\n", "")
+        output_list = [input_string]
+        self.assertEqual(extract_json_objects(input_string), output_list)
```

Comment:
Could we add a test case for full_message_example concatenated to full_message_example?